packages/button/src/Button/Button.styles.ts

```diff
@@ -416,7 +415,7 @@ export function getClassName({
     color,
     fontSize,
     size,
-    { [focus]: usingKeyboard && !disabled },
+    { [focus]: !disabled },
     { [disabledStyle[theme]]: disabled },
   );
 }
```

Comment:
There are a few `&:focus` styles remaining in this file, are we not updating those?

Reply:
I only updated where &:focus styles were conditionally rendered based on usingKeyboard. If :focus styles were supposed to be always visible, I didn't update the selector

Reply:
makes sense